### PR TITLE
added project service to validator

### PR DESCRIPTION
**Pull request by daniel-cit**
Added conversion of project_service to serviceusage  in validator.

[Terrafor-validator](https://github.com/GoogleCloudPlatform/terraform-validator) uses downstream repo [terraform-google-conversion](https://github.com/GoogleCloudPlatform/terraform-google-conversion) to [map](https://github.com/GoogleCloudPlatform/terraform-validator/blob/master/converters/google/mappers.go) the resources that can be converted.

Required step  to fix https://github.com/GoogleCloudPlatform/terraform-validator/issues/153 

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
Added conversion from project_service to CAI service usage for validator
```

### CODE REVIEW

**Comment by ndmckinley** [comment]
Hi there!  To get this to be included in the validator, you need to add this file here: https://github.com/GoogleCloudPlatform/magic-modules/blob/73e8ae08554e2686be4c1fd44a0aba08664a8fa1/provider/terraform_object_library.rb#L76.

**Comment by daniel-cit** [comment]
@ndmckinley Thanks for the info regarding the `terraform_object_library.rb`

**Comment by ndmckinley** [comment]
Great - I don't have any good tests for this code, are you sure it works?  I'll merge it if you're confident it's fully tested.

**Comment by daniel-cit** [comment]
@ndmckinley  **I will add tests for it.**

This was tested using terraform-validator in the context of adding services to an **already existing project**  .

I will add this case and at least another one for a project that does not exist yet.

Do you know of other scenarios that should be considered?

**Comment by ndmckinley** [comment]
I don't - I don't really understand the use cases of that code very well, so I'll trust you if you tell me you've got it under control!  :)

**Comment by ndmckinley** [comment]
Cool, test LGTM.  Is that ready to go in?  I assume you don't have a commit bit so I need to push the button to merge, so just let me know.  :)

**Comment by daniel-cit** [comment]
Hi @ndmckinley, this is ready to go :) 
Thanks for the support with this Pull Request.

**Review by ndmckinley — APPROVED** [review]

**Pull request merged**
